roles/fqcn_migration/defaults/main.yml

```diff
@@ -24,3 +24,4 @@ downstream_placeholder_delete: []
 downstream_placeholder_content: []
 override_galaxy_version: ''
 excluded_roles_from_downstream: []
+case_insensitive_match: false
```

Comment:
Please add the new variable in here as well:
https://github.com/ansible-collections/community.fqcn_migration/blob/main/roles/fqcn_migration/meta/argument_specs.yml